.github/actions/integration-tests/run-ci-stage1

```diff
@@ -242,11 +268,22 @@ run_cmd "podman ps --all --external"
 
 run_cmd "crucible update"
 
-run_cmd "podman ps --all --external"
+case "${CI_ENDPOINT}" in
+    k8s)
+        run_cmd "ssh root@localhost kubectl get nodes"
+
+        run_cmd "ssh root@localhost kubectl get pods"
 
-run_cmd "podman images"
+        run_cmd "microk8s ctr images list"
```

Comment:
this is microk8s specific. If we change to a fedora runner, do we need to change ci again ?

Reply:
The entire thing is heavily focused on the github runner environment using Ubuntu.  There is a *a lot* that would have to change to support a different runner environment...